app/styles/style.css

```diff
@@ -131,4 +137,289 @@ body {
     text-decoration: none;
     display: flex;
   }
+
+  .jobs {
+    padding: 24px;
+  }
```

Comment:
- [ ] Please adhere to the project requirements, as your *previous code reviewer* asked you and suggested you use `grid` for your work section container because it is part of the **requirement** you must have to follow properly. Kindly have a look at the below image.

![image](https://user-images.githubusercontent.com/66476329/235292079-799efcb7-8591-4016-9b30-1f86e057e146.png)

- Actually, as you move further and add more and more features to it, so following the requirement not only make a professional developer who always detail-oriented but also will make your work easier.